deploy/kubernetes/base/resources.yaml

```diff
@@ -648,6 +648,32 @@ webhooks:
       matchLabels:
         juicefs.com/enable-serverless-injection: "true"
 ---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: juicefs-admission-webhook
+webhooks:
+  - name: validate.secret.juicefs.com
+    matchPolicy: Equivalent
+    objectSelector:
+      matchLabels:
+        juicefs.com/validate-secret: "true"
+    rules:
+      - apiGroups:   [""]
+        apiVersions: ["v1"]
+        operations:  ["CREATE", "UPDATE"]
+        resources:   ["secrets"]
+    clientConfig:
+      service:
+        namespace: kube-system
+        name: juicefs-admission-webhook
+        path: "/juicefs/validate-secret"
+      caBundle: CA_BUNDLE
+    timeoutSeconds: 20
```

Comment:
auth requests are expected to finish within something like 5s max, maybe set this to 5?